frontend/demo/src/pages/Inbox/ConversationListItem/index.tsx

```diff
@@ -37,8 +37,9 @@ const mapDispatchToProps = {
 const connector = connect(mapStateToProps, mapDispatchToProps);
 
 const FormattedMessage = ({message}: FormattedMessageProps) => {
-  if (message && message.content[0]) {
-    return <>{message.content[0].text}</>;
+  if (message && message.content) {
```

Comment:
are we using this?

Reply:
yes, think @AudreyKj did the same change in her PR

Reply:
This looks a bit strange though. Why are we not using the render library here, too? :) Because `{text: blah}` is only a valid schema for chatplugin/twilio outbound messages https://gist.github.com/chrismatix/6dd21f96e2dac55b0ff7b3ccc0cd1861

Reply:
Didn't focus to use the renderLibrary here, cause we use styling for the TextRender and if I would use it here, it would have the same style as a normal message. Of course I could have add another boolean which checks if we need a styling here, but was a  bit too off for me for the conversationList